Домашнее задание Модуль №11

Ulyanov-VN opened the pull request.

Ulyanov-VN committed:
Домашнее задание Модуль №11

anastasianadelshtehel commented on app/src/main/java/com/example/playlistmaker/repository/SearchRepository.kt:
```diff
@@ -0,0 +1,9 @@
+import com.example.playlistmaker.network.SearchResponse
+
+class SearchRepository(private val api: ItunesApiService) {
+    suspend fun searchSongs(term: String): SearchResponse =
```
Интересно у тебя уже используются корутины и проект разбит на модули, хотя это темы будущих спринтов...

anastasianadelshtehel commented on app/src/main/java/com/example/playlistmaker/TrackAdapter.kt:
```diff
@@ -10,31 +10,41 @@ import com.example.playlistmaker.R
 }
 
-
+/*fun filter(query: String) {
```
Рекомендация: закомментированный код нужно удалять перед отправкой задания

anastasianadelshtehel reviewed (commented):
Нужно поправить прогресс бар, плейсхолдер и очистку списка после очистки запроса
https://github.com/user-attachments/assets/6471e454-ea8a-4a8d-b10b-2e6dfdcc7355

Ulyanov-VN committed:
Домашнее задание Модуль №11 - устранение замечаний.

1. Исправлен прогресс бар
2. При загрузке треков появляются плейсхолдеры, больше нет белых квадратов
3. Очистка запроса исправлена, список треков удаляется

Анастасия для выполнения домашнего задания, недостаточно информации, которую дает яндекс практикум в своей учебной программе и поэтому 90 процентов информации приходится искать самостоятельно.
